experimental/uefi/baremetal/src/memory.rs

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Portions Copyright © 2019 Intel Corporation.
+
```

Comment:
What does this mean? :) Should those portions be under third_party?

Reply:
Literally that: some portions were copied from rust-hypervisor-firmware (namely, the `RAM_MIN` etc, so about 17 lines of code).

I wrapped them in trivial functions and moved them to third_party.